<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigPowerMeter</class>
 <widget class="QMainWindow" name="ConfigPowerMeter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Device Config Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="instrumentInfoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Instrument: </string>
    </property>
   </widget>
   <widget class="QLabel" name="instrumentAddressLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Address: </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="radioButtonGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>991</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Select Channel/Slot</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>751</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Power Reading</string>
    </property>
    <widget class="QLineEdit" name="powerDisplay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Power:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="powerUnitComboBox">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>Watt</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dBm</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>991</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Adjust Wavelength Settings</string>
    </property>
    <widget class="QLineEdit" name="wavelengthDisplay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="minWavelengthDisplay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="maxWavelengthDisplay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="minWavelengthUnitLabel">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>80</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(unit)</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxWavelengthUnitLabel">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>120</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(unit)</string>
     </property>
    </widget>
    <widget class="QLabel" name="wavelengthLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wavelength:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Min Wavelength:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Max Wavelength:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="wavelengthComboBox">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>nm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>m</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="wavelengthEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>40</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="wavelengthEditUnitLabel">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>40</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(unit)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Configuration</string>
    </property>
    <widget class="QPushButton" name="saveSettingsButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadSettingsButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="saveChangesButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>650</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Submit Changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setNicknameBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set Nickname</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>541</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Integration Time Settings</string>
    </property>
    <widget class="QLabel" name="totalTimeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="periodTimeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Period Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="avgTimeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Averaging Time:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="totalTimeDisplay">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="periodTimeDisplay">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="avgTimeDisplay">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="timeUnitComboBox">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>s</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>min</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ms</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>us</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>hr</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="periodTimeUnitLabel">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>80</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(unit)</string>
     </property>
    </widget>
    <widget class="QLabel" name="avgTimeUnitLabel">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>120</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(unit)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>420</y>
      <width>421</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>MinMax Mode Settings</string>
    </property>
    <widget class="QRadioButton" name="continuousRadioBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Continuous</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="refreshRadioBtn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="dataPointsLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Data Points: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dataPointsEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>instrumentInfoLabel</zorder>
   <zorder>instrumentAddressLabel</zorder>
   <zorder>radioButtonGroupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>saveChangesButton</zorder>
   <zorder>setNicknameBtn</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
